frontend/src/Judge/reducers.js

```diff
@@ -89,8 +88,8 @@ const queue = (state = initialQueueState, action) => {
 // Each show has a queue whose key in the queues state is the show's id
 // Example State:
 // {
-//   1: {order: [1, 2, 3], viewing: 0},
-//   2: {order: [9, 4, 5, 8, 6, 7], viewing: 5}
+//   1: {order: [1, 2, 3], loadingSubmissions: true, loadingVotes: false},
+//   2: {order: [9, 4, 5, 8, 6, 7], loadingSubmissions: true, loadingVotes: false}
```

Comment:
Also noticed the values in `order` in these comments should be made into strings – put single quotes around them.